sdk/testing/server/config.go

```diff
@@ -31,29 +31,43 @@ type Config struct {
 }
 
 // Option a function that wish to modify a given kcp configuration.
-type Option func(*Config) *Config
```

Comment:
For my understanding - what about this was blocking external users?

Reply:
It wasn't. But it's also unnecessary and I have seen the non-functional style elsewhere.

Reply:
Note: this code was not exposed before. So we are just cleaning up stuff on the go.

Reply:
Makes sense, thanks!